source/gui/frmhelpgeneratorunit.pas

```diff
@@ -1069,6 +1071,7 @@ procedure TfrmHelpGenerator.ButtonGenerateDocsClick(Sender: TObject);
     end;
 
     if PasDoc1.Generator is TGenericHTMLDocGenerator then
+      if CheckOpenHTML.Checked then
       OpenURL(HtmlDocGenerator.DestinationDirectory + 'index.html');
```

Comment:
The line `OpenURL(...)` should be indented by 2 additional spaces, since it is now under `if xxx` condition.

Reply:
Fixed by rebasing and rewriting because of merge conflict